docs/rules/space-before-keywords.md

```diff
@@ -0,0 +1,53 @@
+# Require or disallow spaces before keywords (space-before-keywords)
+
+Some styleguides require or disallow spaces preceding certain keywords.
+
+## Rule Details
+
+This rule will enforce consistency of spacing before the keywords `if`, `else`, `for`,
+`while`, `do`, `switch`, `throw`, `try`, `catch`, `finally`, `with`, `break`, `continue`,
+`return`, `function`, `yield`, `class`, `super` and label statements.
+
+This rule accepts one argument: `"always"` or `"never"`. If `"always"` is used, the keywords
+must be followed by at least one space. If set to `"never"`, no spaces will be allowed before
+the keywords `else`, `while` (do...while), `finally` and `catch`. The default value is `"always"`.
+
+The following patterns are considered warnings:
+
+```js
+if (foo) {}else {}
+```
```

Comment:
Hi, we've started putting configuration comments and expected errors right inline with the example code.  Take a look at http://eslint.org/docs/rules/comma-dangle for an example of this.  If you could follow this same style and put the comments in this rule, it would save me from having to put it in later.  I'm currently going through all 170-ish rules and one fewer to deal with would be nice.  :)  If you have questions about it, please let me know.